src/YahooFinance.py

```diff
@@ -198,9 +198,44 @@ def parse_modules(self, content):
     results = data.get('quoteSummary', {}).get('result', None)
     if not results:
       logging.error('Could not parse response for url: ' + self.url)
-      return
+      return False
     for module in self.modules:
       for result in results:
         if module in result:
           self.module_data[module] = result[module]
-          break
\ No newline at end of file
+          break
+    return True
+
+  def _get_balance_sheet_history(self, position):
+    history = []
+    for stmt in self.module_data.get('balanceSheetHistory').get('balanceSheetStatements'):
+      history.append(stmt.get(position).get('raw'))
+    return history
+
+  def _get_income_statement_history(self, position):
```

Comment:
[Same as above]

suggestion: instead of 'position' can we call this 'key'? since it's used to index the dictionary of data?

Or was `position` terminology used somewhere else in the data?

Reply:
I thought about it as a position in balance sheet or income statement, "paper" version :). `key` is more meaningful from the python dict perspective. However, I'll change that.

Reply:
Changed.